### DISCUSSION TITLE
💡: Androidの戻るボタンやiOSのスワイプでの画面遷移の考慮が必要なことを書いたほうがいいか？

### DISCUSSION THREAD

**Discussion by tani-takanori** [ws-4020/mobile-app-crib-notes#244] [General]
画面遷移が複雑になったときアプリのUIコンポーネント外の遷移（例：Androidのボタン）が発生したときの考慮が漏れることがあるようなら「ハマりがちな落とし穴」などに記載しておいたほうがいいか？

もしくは設計した情報として、ExampleApplicationのなかに入れておくのが良いか。

以下の選択があるかなと思っている

- 言及は不要
- ReactNativeのハマりがちな落とし穴に残す
- ExampleApplicationで設計
- その他

**Comment by ghost** [comment]
> 言及は不要

**Comment by ghost** [comment]
> React Nativeのハマりがちな落とし穴に残す

**Comment by ghost** [comment]
> Example Applicationで設計

**Comment by ghost** [comment]
> その他

**Comment by ghost** [comment]
こうすれば投票できそう

**Reply by tani-takanori** [reply]
おー
